### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by hosjiu1702** [googlecodelabs/feedback#780]
https://codelabs.developers.google.com/codelabs/cloud-tensorflow-mnist/#2

> How do we know if the trained neural network performs well or not? Using the training dataset to test the network would be cheating. It has already seen that dataset multiple times during training and is most certainly very performant on it. We need another labeled dataset, never seen during training, to evaluate the "real-world" performance of the network. It is called an **"validation dataset"**

`Validation dataset` should be `"Test dataset"`?

Because `validation set` is used to choose your learning algorithm and parameters.
And `test set` is almost used to evaluate your model with "real world" data?

Reference: https://www.deeplearning.ai/machine-learning-yearning/ (**page 15**)
